app.js

```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
+    div.setAttribute('id', `${i}`);
+    div.innerHTML = `
+   <div class="card-left-mobile">
+   <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   <div class="card-right">
+     <h2>${portfolioItemsDetail[i].name}</h2>
+     <ul class="cano">
+       <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+       <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+       <li>2015</li>
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
+     </ul>
+     <button class="btn-default portfolio-items-detail">See project</button>
+   </div>
+   <div class="card-left-desk">
+     <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   `;
+    portfolio.appendChild(div);
+  }
+}
+const portfolioItems = document.querySelectorAll('.portfolio-items-detail');
+if (portfolioItems) {
+  portfolioItems.forEach((cards) => {
+    cards.addEventListener('click', (e) => {
+      const idValue = e.target.closest('.card').id;
+      for (let i = 0; i < portfolioItemsDetail.length; i += 1) {
+        if (portfolioItemsDetail[i].id === idValue) {
+          const popupPage = document.createElement('page');
+          popupPage.classList.add('popup-page');
+          // console.log(porfolioItemsDetail[i].name);
+          popupPage.innerHTML = `
+          <div class="background"></div>
+          <div class="card flex-main">
+            <button class="close-btn" type="submit">
+              <i class="fas fa-times"></i>
+            </button>
+            <div class="card-right">
+              <h2>${portfolioItemsDetail[i].name}</h2>
+              <ul class="cano">
+                <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].year}</li>
+              </ul>
+              <div class="snap-image">
+                <img src="${portfolioItemsDetail[i].images}" alt="${portfolioItemsDetail[i].name}">
+              </div>
+              <div class="flex-nl">
+                <p>${portfolioItemsDetail[i].text}</p>  
+                <ul class="lang">
+                ${portfolioItemsDetail[i].workStack.map((stack) => `<li>${stack}</li>`).join('')}
+                </ul>
+                  <div class="link-button">
+                    <button class="btn-default"><a href="${portfolioItemsDetail[i].live} class="see-btn">See live<img
+                    src="images/live.png" alt="live-link" /></a></button>
+                    <button class="btn-default"><a href="${portfolioItemsDetail[i].source} class="see-btn">See source<img
+                    src="images/Vector.png" alt="live-link" /></a></button>
+                  </div>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] As suggested by previous code reviewers, kindly invest quality time in ensuring your popup is consistent with the Figma version. Please check this video to see how things are at the moment.

[scrnli_29_07_2023_23-49-48.webm](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/99715640/95db2c24-eb5e-41b8-9c71-8e6dcd839ea7)

  - Please patiently work on the responsiveness of the popup and ensure that both the mobile and desktop versions are as we have it on Figma.

Reply:
i am out of time!